x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/index.ts

```diff
@@ -11,6 +11,8 @@ export default function aiAssistantApiIntegrationTests({
   loadTestFile,
 }: DeploymentAgnosticFtrProviderContext) {
   describe('observability AI Assistant', function () {
+    // see details: https://github.com/elastic/kibana/issues/220513
+    this.tags(['failsOnMKI']);
```

Comment:
The tests that can't run on MKI are already skipped. Which ones are failing, and not skipped already?